# configuration for FDG set up


# Monitored door sensor name (same than in OpenHAB or Mongo)
sensorName: 'Entry_Door_Contact'


# Report door open after 30 seconds
t_alarm: 30

# Topic to publish alarms to. It is an string like this:
# "Alarm:[sensorName]:[seconds opened]
notifTopicName: '/ais_notifications'


        
